ga_resources/tasks.py

```diff
@@ -7,7 +7,7 @@
 from django.utils.timezone import utc
 
 
-@task(ignore_result=True)
+#@task(ignore_result=True)
 def refresh_resource(pk):
```

Comment:
This module is not needed since we are not using GeoAnalytics (GA). However, a few utility functions and template tag functions are used throughout our HydroShare code base, so it will be enough of cleanup work by itself to remove this app and move relevant functions into hs_core. However, since these celery tasks are crontab tasks, once I set up celery beat service to run periodic tasks, these tasks are put into the queue, which raises errors and exceptions, which is why I comment them out so that these functions will not be put into the queue to be scheduled. Ideally, this whole app should be removed as part of cleanup work, but I think it should be done as a subsequent issue rather than doing it in this feature branch due to its widespread usage throughout the code base.  I have verified these non-used tasks not run at all after I comment out the @task decorators, so it should be fine at this point. I will create an issue to remove this app as part of clean up to tackle this later.